libkbfs/md_util.go

```diff
@@ -507,7 +507,9 @@ func decryptMDPrivateData(ctx context.Context, codec kbfscodec.Codec,
 		}
 	}
 
-	// Re-embed the block changes if it's needed.
+	// Re-embed the block changes if it's needed.  TODO: we don't need
+	// to do this in minimal mode, since there's no node cache
```

Comment:
I must be missing something, but why does not having a node cache imply not neededing to re-embed the block changes, in particular? Isn't it just the fact that we don't do any writes?